src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -48,6 +48,7 @@ using namespace app::Clusters::OtaSoftwareUpdateRequestor::Structs;
 constexpr uint32_t kDelayQueryUponCommissioningSec = 30; // Delay before sending the initial image query after commissioning
 constexpr uint32_t kImmediateStartDelaySec         = 1;  // Delay before sending a query in response to UrgentUpdateAvailable
 constexpr System::Clock::Seconds32 kDefaultDelayedActionTime = System::Clock::Seconds32(120);
+constexpr uint32_t kDelayNotifyUpdateApplied                 = 10; // Delay before sending notify update applied command
```

Comment:
Can this be made configurable? I think in most cases the delay is not really needed and, so a shorter delay will speed up smoke-testing.

Reply:
nit: kDelayNotifyUpdateApplied -> kDelayNotifyUpdateAppliedSec